Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM amazonlinux:2023
+
+RUN yum update -y & yum install -y yum-utils shadow-utils gcc libffi-devel python3 python3-pip openssh-clients rsync
+
+ENV ANSIBLE_HOST_KEY_CHECKING=False
+
+RUN echo "Installing Ansible"; \
```

Comment:
and docker

Reply:
Of course! I will change it now.

It is in a separate repo so that we can keep the `itu-minitwit` repo clean. We can still reference this github action from other repos in our organization. I will add some changes in `itu-minitwit` later.